src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -619,6 +619,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'default': true,
 				'description': localize('tips.enabled', "When enabled, will show the watermark tips when no editor is open.")
 			},
+			'workbench.tabIndex.enabled': {
```

Comment:
setting name should align with other setting names. Something like `workbench.editor.showTabIndex`